src/main/java/seedu/address/logic/parser/ArgumentMultimap.java

```diff
@@ -75,4 +76,24 @@ public void verifyNoDuplicatePrefixesFor(Prefix... prefixes) throws ParseExcepti
             throw new ParseException(Messages.getErrorMessageForDuplicatePrefixes(duplicatedPrefixes));
         }
     }
+
+    /**
+     * Returns the number of arguments.
+     */
+    public int getLength() {
+        return argMultimap.size();
+    }
+
+    /**
+     * Returns the single prefix used in the argument. Assumes that only one prefix is used.
+     * If more than one prefix is used, an exception is thrown.
+     */
+    public Prefix getSinglePrefix() throws ParseException {
+        if (argMultimap.size() != 2) {
```

Comment:
I like this exception as it accurately reflects the requirement for the sort command - we can only sort by one category